crux-mir/.travis.yml

```diff
@@ -0,0 +1,100 @@
+dist: xenial
```

Comment:
Probably should remove this `.travis.yml`

Reply:
Is there an easy way to keep the crux-mir CI running after the merge?  I'd rather not disable it if we can avoid it

Reply:
I don't see why not; it should just move into the top level travis config, or get ported to github actions.

Reply:
I'll look at moving/porting it. I want to set up something for producing binaries people can download for the release, and have already set up an Action for that for `crux-llvm`.

Reply:
I think I'm going to do this as part of a separate PR, to avoid a messy merge. So this PR is now actually ready for review.